gradle.properties

```diff
@@ -38,9 +38,7 @@ jerseyVersion=2.27
 junitVersion=4.12
 lombokVersion=1.18.4
 marathonClientVersion=0.6.0
-metrics3StatsdVersion=4.2.0
-metricsVersion=3.2.5
-metricsGangliaVersion=1.0.10
+micrometerVersion=1.1.2
```

Comment:
Just noticed that `1.1.3` has been released. Maybe it is worth to consider before merging the PR, in case that it does not involve code changes.

Reply:
fixed